packages/jaeger-ui/src/components/SearchTracePage/SearchForm.js

```diff
@@ -34,7 +34,7 @@ import * as jaegerApiActions from '../../actions/jaeger-api';
 import { formatDate, formatTime } from '../../utils/date';
 import reduxFormFieldAdapter from '../../utils/redux-form-field-adapter';
 import { DEFAULT_OPERATION, DEFAULT_LIMIT, DEFAULT_LOOKBACK } from '../../constants/search-form';
-import getConfigValue from '../../utils/config/get-config';
+import { getConfigValue } from '../../utils/config/get-config';
```

Comment:
should there be a unit test that would catch this?

Reply:
Also, here maybe problem with build, package, dockerize etc. Because I did not see this deleted 37th row, when I am debugging in Chrome. I suggest testing after building into docker.

Reply:
I'll add tests.

Reply:
Docker image contains JS transpiled to ES5. I'd assume the code to look very different there (also minified).

Reply:
![image](https://user-images.githubusercontent.com/1571433/75424687-39cabe00-5952-11ea-9c68-62ed54d7829f.png)
screenshot from chrome devtools.